### ISSUE TITLE
[Apollo 1.12] Multiple Notifications

### ISSUE THREAD

**Issue by serialtoon** [christianselig/apollo-bugs#1583]
Hello everyone, after the latest update (1.12) i have been receiving duplicate notifications and i was wondering if anyone else has the same issue. Checking on Reddit.com, i only see the one notification where Apollo notifies me twice in quick succession and the messages panel also displays 2 of the same message.

App Version: 1.12

iOS version: 15.2

Device Type: iPhone 13 Pro Max

How often can you reproduce the issue: Every time i get a new message

Reproducible Steps: Just receive notification from a message on Reddit.

Issue with replies available on r/ApolloApp - https://old.reddit.com/r/apolloapp/comments/rqjugt/duplicate_notifications_apollo_ultra_on_112/

![image](https://user-images.githubusercontent.com/38597931/148459409-258e42f5-a971-45e9-ab07-cda919637e4e.png)

**Comment by cristian-rivera** [comment]
Can confirm this is happening to me as of tonight! Seems to be exactly every 2 minutes more or less like yours.

**Comment by goodboydheeraj** [comment]
Happening to me as well on latest version of the app on iPhone X & 7 Plus both on iOS 14.8.

https://imgur.com/a/CjmlYUU

**Comment by hungrypigright** [comment]
I am having the same issue as well. Seems like I receive 5 notifications each time.

**Comment by wiffleman** [comment]
Yep also having this issue. 3-4 notifications per comment/message
